quantum/quantum.c

```diff
@@ -330,6 +339,10 @@ bool process_record_quantum(keyrecord_t *record) {
                 return false;
             case QK_CLEAR_EEPROM:
                 eeconfig_init();
+                soft_reset_keyboard();
+                return false;
+            case QK_REBOOT:
```

Comment:
This block should probably be moved under `QK_BOOTLOADER` within the `#ifndef NO_RESET`